db/migrate/20190901194835_create_items.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateItems < ActiveRecord::Migration[6.0] # :nodoc:
+  def change
+    create_table :items do |t|
+      t.uuid :uuid, null: false
+      t.string :item_number, null: false
+      t.string :item_description
+      t.integer :is_active, default: 1, null: false
+      t.timestamps
+
+      t.index :uuid, unique: true
```

Comment:
These could be written more succinctly in the column definitions as:

```ruby
t.uuid :uuid, index: { unique: true }, null: false
t.string :item_number, index: { unique: true }, null: false
```

Reply:
Fixed